Podcast/BrowseTopicsViewController.swift

```diff
@@ -0,0 +1,81 @@
+//
+//  BrowseTopicsViewController.swift
+//  Podcast
+//
+//  Created by Mindy Lou on 12/27/17.
+//  Copyright © 2017 Cornell App Development. All rights reserved.
+//
+
+import UIKit
+
+/// Displays a list of topics from either the DiscoverViewController or from a parent BrowseTopicsViewController.
+class BrowseTopicsViewController: ViewController, UITableViewDelegate, UITableViewDataSource {
+
+    let reuseIdentifier = "Reuse"
+    let rowHeight: CGFloat = 54
+
+    var topics: [Topic] = []
+    var topicsTableView: UITableView!
+
+    var parentTopic: Topic? // will be non-nil if currently showing "See All" and subtopics
+
+    convenience init(parent: Topic) {
+        self.init()
+        parentTopic = parent
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = .offWhite
```

Comment:
i'd say make it the same color as the topic cells

Reply:
so like the pink, blue, teal, yellow?